packages/quantic/force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -1106,4 +1106,67 @@
       <protected>false</protected>
       <shortDescription>Opens in the browser tab</shortDescription>
     </labels>
+    <labels>
+        <fullName>quantic_ComponentError</fullName>
+        <value>{{0}} component error</value>
+        <language>en_US</language>
+        <protected>false</protected>
+        <shortDescription>c-quantic-pager component error</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_LookAtDeveloperConsole</fullName>
+      <value>Look at the developer console for more information.</value>
+      <language>en_US</language>
+      <protected>false</protected>
+      <shortDescription>Look at the developer console</shortDescription>
+    </labels>
+    <labels>
+      <fullName>quantic_InvalidMaxNumberOfDocumentSuggestions</fullName>
+      <value>The maximum number of document suggestions must be an integer be greater than 0.</value>
```

Comment:
@mmitiche Do you mean...?

"<value>The maximum number of document suggestions must be an **integer greater** than 0.</value>"

Reply:
yess exactly thank you for catching that 🙌